spec/requests/vendor_api/get_spec_spec.rb

```diff
@@ -0,0 +1,9 @@
+require 'rails_helper'
+
+RSpec.describe 'Vendor API - GET /api/v1/spec.yml', type: :request do
+  it 'returns the spec in YAML format' do
+    get '/api/v1/spec.yml'
+
+    expect(response).to have_http_status(200)
```

Comment:
My 200 is empty:

```
 % curl -Iv localhost:3000/api/v1/spec.yml                                          [1357-merge-tech-docs]
*   Trying ::1...
* TCP_NODELAY set
* Connected to localhost (::1) port 3000 (#0)
> HEAD /api/v1/spec.yml HTTP/1.1
> Host: localhost:3000
> User-Agent: curl/7.54.0
> Accept: */*
>
< HTTP/1.1 200 OK
HTTP/1.1 200 OK
< X-Frame-Options: SAMEORIGIN
X-Frame-Options: SAMEORIGIN
< X-XSS-Protection: 1; mode=block
X-XSS-Protection: 1; mode=block
< X-Content-Type-Options: nosniff
X-Content-Type-Options: nosniff
< X-Download-Options: noopen
X-Download-Options: noopen
< X-Permitted-Cross-Domain-Policies: none
X-Permitted-Cross-Domain-Policies: none
< Referrer-Policy: strict-origin-when-cross-origin
Referrer-Policy: strict-origin-when-cross-origin
< Access-Control-Allow-Origin: *
Access-Control-Allow-Origin: *
< Content-Type: text/yaml; charset=utf-8
Content-Type: text/yaml; charset=utf-8
< ETag: W/"36a9e7f1c95b82ffb99743e0c5c4ce95"
ETag: W/"36a9e7f1c95b82ffb99743e0c5c4ce95"
< Cache-Control: max-age=0, private, must-revalidate
Cache-Control: max-age=0, private, must-revalidate
< X-Request-Id: eea52356-2fe1-43b8-a95a-75d3f0ded2cc
X-Request-Id: eea52356-2fe1-43b8-a95a-75d3f0ded2cc
< X-Runtime: 0.003908
X-Runtime: 0.003908

<
* Connection #0 to host localhost left intact
```

Reply:
Fixed https://github.com/DFE-Digital/apply-for-postgraduate-teacher-training/pull/855/commits/0a228141a9ec9a7704894c9809e1c5ba96af1113!